src/components/common/Modal.vue

```diff
@@ -39,38 +39,24 @@
             </div>
```

Comment:
Can you also change the modal behavior so that if you click around the modal, it will close?

Reply:
Fixed
https://gyazo.com/47a17553e334b9a3bd905489781fd307